### ISSUE TITLE
Blended Mobility

### ISSUE THREAD

**Issue by Belenchusky** [erasmus-without-paper/ewp-specs-api-iias#55]
We are developing the new IIAS template right now and we have a problem with the blended field. This field is at the agreement level,neverthelesin the GET API the same filed  operate for each Student Mobility for Studies.

That's a contradiction, isn´t it?

![imagen](https://user-images.githubusercontent.com/47413457/119473436-72bb3380-bd4b-11eb-868b-561166fbd65e.png)
![imagen](https://user-images.githubusercontent.com/47413457/119473570-954d4c80-bd4b-11eb-8c2e-5941756e298e.png)

**Comment by kamil-olszewski-uw** [comment]
During the adaptation of IIAs API to the rules of the new Erasmus+ edition, a design decision was made (with the knowledge and consent of the EUF) that the "Blended" field would be at the cooperation conditions level, which would allow for more clarity and less surprises.

**Comment by Belenchusky** [comment]
I agree with you, but who is in charge of bringing the decision made in the API to the template? I just checked it online and the API mismatch still exists.

**Comment by lioliosn** [comment]
There is no contradiction. The blended option is not at the agreement level as it refers only to students and not to staff mobilities. Therefore, the API connected more effectively this condition to students' mobilities. It is the template's business logic that it is translated to API specs.

**Comment by Belenchusky** [comment]
@lioliosn I see your point of view but that implies that this condition affects all student mobilities that you have in the same agreement and that is not always the case.

Let me show a real agreement signed through EWP between our university and P BEJA 01. As you can see, I had to change the template to be able to represent the information that is obtained from the GET API.

![imagen](https://user-images.githubusercontent.com/47413457/166904441-7115fe7a-0025-4622-8b8a-17bdd5fe9b23.png)

**Comment by lioliosn** [comment]
@Belenchusky if you have a look at the API specs, it fits exactly your use case. You have blended mobiliyty option for every mobility flow per subject area code. I understand that your problem is the template, but the API has solved this for you.

**Comment by Belenchusky** [comment]
@lioliosn Yes, the problem is the official IIA template. As you can see I had to modify it on my own.
